dev-docs/bidders/bidtheatre.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: bidder
+title: Bidtheatre
+description: Bidtheatre Prebid Bidder Adapter
+biddercode: bidtheatre
+gvl_id: 30
+tcfeu_supported: true
+media_types: banner, video
+safeframes_ok: true
+deals_supported: true
+floors_supported: true
+fpd_supported: true
+ortb_blocking_supported: partial
+multiformat_supported: will-bid-on-any
+privacy_sandbox: topics
+pbjs: true
+sidebarType: 1
+pbs: true
+pbs_app_supported: true
+---
+
+### Registration
```

Comment:
```suggestion
### Registration

```

Weirdly markdownlint is not triggered. There must always be a new line after a heading